### ISSUE TITLE
Readme: clarify purpose and features of org-linkz

### ISSUE THREAD

**Issue by novoid** [p-kolacz/org-linkz#8]
Hi,

I'd like to have the readme file changed such that I learn what to expect from org-linkz. Open questions that came to my mind that did not get answered by the readme so far:

- Which browsers are supported?
- Which operating system is supported?
- Since you are stating "Manage browser bookmarks in org file" I would assume that any change of that Org file gets then synced back to the browser bookmarks. Is this correct?
  - If yes: 
    - When do the changes get synced back? When Org file gets saved? With a command?
    - What about resolving sync conflicts when the browser is adding a new bookmark while I'm modifying the Org file?
    - Does the Org file gets updated when the browser changes a bookmark in the background?
  - If no:
    - What do you mean by "manage" then? Is it just a one-time, uni-directional conversion to Org? (If so, I'd rephrase the description since it does not involve "management" to my understanding)

Thanks for your cool project and for clarifying!

**Comment by aspiers** [comment]
I agree, the purpose of the project was not entirely clear to me on first read either.  Does look interesting though!  You may also be interested in https://github.com/aspiers/desktop-config/tree/master/lib/browser/userscripts/page-id-helpers which seems to have some similarities, albeit a wider scope than just emacs and org-mode.
